packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/steps/Anciennete/store/__tests__/store.test.ts

```diff
@@ -0,0 +1,195 @@
+import { loadPublicodesRules } from "../../../../../api";
+import { createIndemniteLicenciementStore } from "../../../../store";
+import { MOTIFS } from "../../components/AbsencePeriods";
+
+describe("Ancienneté store", () => {
+  let store = createIndemniteLicenciementStore(
+    loadPublicodesRules("indemnite-licenciement")
+  );
+
+  // to reset store after each unit test
+  afterEach(() => {
+    store = createIndemniteLicenciementStore(
+      loadPublicodesRules("indemnite-licenciement")
+    );
+  });
```

Comment:
Une manière d'écrire qui serait peut être plus élégante (pour éviter d'avoir 2 fois le create)
```suggestion
  let store;
beforeEach(() => {
store = createIndemniteLicenciementStore(
      loadPublicodesRules("indemnite-licenciement")
    );
})
```

Reply:
Oui exact, j'avais fait ça pour l'auto typage 😅